engine/classes/ElggPluginPackage.php

```diff
@@ -234,7 +237,7 @@ private function hasReadableConfigFile() {
 		}
 
 		$this->errorMsg =
-			_elgg_services()->translator->translate('ElggPluginPackage:InvalidPlugin:UnreadableConfig');
+			elgg_echo('ElggPluginPackage:InvalidPlugin:UnreadableConfig');
```

Comment:
Is it possible to keep the `elgg_echo` on the same line a `$this->errorMsg`? now that the line is shorter.....